style.css

```diff
@@ -0,0 +1,816 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+html {
+  overflow-x: hidden;
+}
+
+ul {
+  list-style: none;
+}
+
+body {
+  background-color: #f4f5f6;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* desktop navbar */
+
+.top_nav {
+  background-color: #f72210;
+  height: 95px;
+}
+
+img:hover {
+  cursor: pointer;
+}
+
+.top_nav > div > img {
+  width: 366px;
+  height: 85px;
+  cursor: pointer;
+}
+
+.top_nav > div {
+  margin-bottom: 10px;
+}
+
+.top_mar {
+  margin-right: 20px;
+}
+
+.top_nav > p {
+  color: white;
+  font-size: 15px;
+}
+
+#go_right {
+  margin-left: 15px;
+}
+
+h4 > a {
+  font-weight: bold;
+  color: #a0a0a0;
+  font-size: 20px;
+}
+
+h6 > a {
+  font-weight: bold;
+  color: black;
+  font-size: 24px;
+}
+
+h5 > a {
+  color: black;
+  font-weight: bold;
+  font-size: 17px;
+}
+
+.in_images > a {
+  margin-bottom: 0;
+  padding: 5px 15px 6px;
+  background-color: rgba(255, 255, 255, 0.9);
+  font-weight: bold;
```

Comment:
Create a new class for `font-weight: bold;`. Then add this class to all the elements that need it in the HTML file. Let's keep the DRY rule respected.